symplyphysics/definitions/volumetric_coefficient_of_thermal_expansion.py

```diff
@@ -0,0 +1,64 @@
+from sympy import Eq, Derivative
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+    symbols,
+)
+
+# Description
+## The coefficient of thermal expansion describes how the size of an object changes with a change in temperature
+## at constant pressure.
+
+# Law: alpha_V = (dV/dT)_p / V
+## alpha_V - volumetric coefficient of thermal expansion
+## V - volume of body (gas, liquid or solid)
```

Comment:
We either should take volume at some temperature point, or we'll result in coefficient as a function.

Reply:
Yeah here it should be `(dV/dT)_p / V(T)`